package.json

```diff
@@ -113,8 +113,8 @@
     "tippy.js": "^6.3.1",
     "uikit": "3.5.16",
     "url-loader": "^4.1.1",
+    "v-calendar": "^2.3.2",
     "vue": "^2.6.11",
-    "vue-datetime": "^1.0.0-beta.10",
```

Comment:
I think there's a bunch of `vue-datetime` dependencies we can get rid of, too (e.g. weekstart). Could you take a look and update them accordingly?

Reply:
Removed weekstart, luxon seemed to come with vue-datetime back in the days as well but we're using it outside of the datepicker by now